TrackingTools/TrajectoryFiltering/interface/CompositeTrajectoryFilter.h

```diff
@@ -26,6 +27,10 @@ class CompositeTrajectoryFilter : public TrajectoryFilter {
 
   ~CompositeTrajectoryFilter() override {}
 
+  static void fillPSetDescription(edm::ParameterSetDescription& iDesc){
+    // !!! MISSING
```

Comment:
Just highlighting this function, because it needs to be implemented.

Reply:
yeah, I noticed that, but I was more curious about the ones that I don't understand, before tackling the one I do understand :)